apps/aecore/test/aecore_tx_test.exs

```diff
@@ -73,4 +79,29 @@ defmodule AecoreTxTest do
 
     assert !SignedTx.is_coinbase?(signed_tx)
   end
+
+  test "invalid spend transaction", tx do
+    from_acc = Wallet.get_public_key()
+    value = 500
+    fee = 300
+
+    :ok = Miner.mine_sync_block_to_chain()
+
+    assert Enum.count(Chain.chain_state().accounts) == 1
+    assert Chain.chain_state().accounts[Wallet.get_public_key()].balance == 100
+
+    payload = %{to_acc: tx.to_acc, value: value}
+    tx_data = DataTx.init(SpendTx, payload, from_acc, fee, tx.nonce)
+
+    priv_key = Wallet.get_private_key()
+    {:ok, signed_tx} = SignedTx.sign_tx(tx_data, priv_key)
+
+    :ok = Pool.add_transaction(signed_tx)
+
+    :ok = Miner.mine_sync_block_to_chain()
+
+    # We should have only made two coinbase transactions
+    assert Enum.count(Chain.chain_state().accounts) == 1
+    assert Chain.chain_state().accounts[Wallet.get_public_key()].balance == 200
```

Comment:
can you reduce value to 200 and fee to 50, mine one more block, it should be included by then

Reply:
@thepiwo I should mine 2 more blocks, because the miner will need 300 tokens so that the transaction is valid, so the transaction will be mined on the 4th mined block

Reply:
yes realized that, good!